python/cuml/cuml_accel_tests/upstream/umap/xfail-list.yaml

```diff
@@ -14,7 +14,6 @@
   - "umap.tests.test_umap_ops::test_disconnected_data[True-hellinger-1]"
   - "umap.tests.test_umap_ops::test_disconnected_data[True-hellinger-5]"
   - "umap.tests.test_umap_ops::test_umap_graph_layout"
-  - "umap.tests.test_umap_ops::test_umap_transform_embedding_stability"
```

Comment:
This one is no longer erroring for me locally, but I'm not sure why. Going to let CI cycle before debugging further.